rust/kernel/time.rs

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: GPL-2.0
+
+//! This module contains timing methods for functionality like sleeping.
+
+use core::ffi::c_uint;
+
+use crate::bindings::msleep as msleep_bindgen;
```

Comment:
Please use `bindings::msleep` for the call, it avoids the rebinding `use` declaration (see other `bindings::` uses).

This also helps to avoid having raw C names in the Rust "namespace", which may be surprising.